app/src/main/kotlin/com/intive/picover/main/view/MainActivity.kt

```diff
@@ -75,54 +74,55 @@ class MainActivity : ComponentActivity() {
 			NavigationItem.Camera,
 			NavigationItem.Profile,
 		)
-		Scaffold {
-			AnimatedVisibility(
-				visible = navigationType == NavigationType.NAVIGATION_BAR,
-			) {
-				val coroutineScope = rememberCoroutineScope()
-				val drawerState = rememberDrawerState(initialValue = DrawerValue.Closed)
-				PicoverNavigationBar(
-					items = navigationItems,
-					navController = navController,
-					onItemClick = {
-						if (it is NavigationItem.Profile) {
-							coroutineScope.launch { drawerState.open() }
-						} else {
-							coroutineScope.launch { drawerState.close() }
-							navController.navigateWithSingleTop(it)
-						}
-					},
-					modifier = Modifier.padding(it),
-					drawerState = drawerState,
-				)
-			}
-			AnimatedVisibility(
-				visible = navigationType == NavigationType.NAVIGATION_RAIL,
-			) {
-				PicoverNavigationRail(
-					items = navigationItems,
-					navController = navController,
-					onItemClick = {
-						navController.navigateWithSingleTop(it)
-					},
-					modifier = Modifier.padding(it),
-				)
-			}
-			AnimatedVisibility(
-				visible = navigationType == NavigationType.NAVIGATION_DRAWER,
-			) {
-				PicoverNavigationDrawer(
-					items = navigationItems,
-					navController = navController,
-					onItemClick = {
-						navController.navigateWithSingleTop(it)
-					},
-					modifier = Modifier.padding(it),
-				)
+		Scaffold { paddingValues ->
+			if (navigationType == NavigationType.BOTTOM_BAR) {
```

Comment:
Here you should use `when` to due to its exhaustive feature

Reply:
Done